data/overlayfiles/rancher-deployer/cnab/app/charts/rancher/values.yaml

```diff
@@ -106,15 +106,10 @@ privateCA: false
 noProxy: 127.0.0.0/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16,.svc,.cluster.local
 
 # Override rancher image location for Air Gap installs
-rancherImage: registry.rancher.com/rancher/rancher
+rancherImage: rancher/rancher
```

Comment:
shouldn't the registry still be registry.rancher.com?

Reply:
restored